resources/v1/clusters/testing/fixtures/get-filtered.json

```diff
@@ -0,0 +1,30 @@
+{
+  "cluster": {
+    "id": "pakistan",
+    "services": [
+      {
+        "shared": true,
```

Comment:
All services have `shared = false` now, and because of `omitempty`, we don't report this field at all anymore. Finding this field in the fixture here made me realize that I forgot to remove the obsolete field in Limes, [which I just now did](https://github.com/sapcc/limes/commit/67be9868d56e9f0fdc5cd36e37e88005b1fb4ec5). Please remove the `shared` field from the fixtures here.